test/Nancy.Tests/Unit/Sessions/DefaultSessionObjectFormatterFixture.cs

```diff
@@ -68,11 +72,16 @@ public void Should_return_null_when_deserializing_empty_string()
         [Serializable]
         public class Payload : IEquatable<Payload>
         {
-            public int IntValue { get; private set; }
+            public int IntValue { get;  set; }
+
+            public bool BoolValue { get;  set; }
 
-            public bool BoolValue { get; private set; }
+            public string StringValue { get;  set; }
 
-            public string StringValue { get; private set; }
+            public Payload()

```

Comment:
> Its a test class used for testing!. Serialization needs an empty constructor

Sure, in this case. But people use this in production, right? This should reflect production code. Which means if you had to add it, other people will probably have to add it as well. So this is a breaking change.